src/translations/enTranslations.json

```diff
@@ -430,7 +430,6 @@
       "contributor_required": "You need to register at least one contributor for this registration",
       "editor_required": "You need to register at least one editor for this registration",
       "employments_missing_data": "Every employment must have position, position percent and start date",
```

Comment:
merge conflict som kommer her er fra endringer i Lokalise av oversetter. Skal være "job percentage" i stedet for "position percent" her